tictactoe/Resources.resx

```diff
@@ -0,0 +1,54 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
+    <resheader name="resmimetype">
+        <value>text/microsoft-resx</value>
+    </resheader>
```

Comment:
maybe do Resources.en.resx?

Reply:
@ssunday I tried to move out the data key values  from Resources.resx to Resources.en.resx , this throws a build error. So both files look exactly the same right now (804f2d2).